apps/pixeloven/site/src/pages/index.js

```diff
@@ -86,7 +86,7 @@ function Home() {
                 To create a project called <i>my-app</i>, run this command:
               </p>
               <CodeBlock className="language-sh">
-                npx pixeloven init my-app
+                npx @pixeloven/cli init
```

Comment:
This won't work until I can version the current CLI. Hoping all this work makes the CLI a bit faster to cache via NPX